.editorconfig

```diff
@@ -0,0 +1,31 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,py}]
```

Comment:
Do you need `py` files? Or `Makefile`? :smile: 

Please try to make this `.editorconfig` as simple as possible and remove any stuff unrelated to JS

Reply:
Done